server/loginserver.ts

```diff
@@ -223,10 +223,14 @@ const LoginServer = Object.assign(new LoginServerInstance(), {
 });
 
 FS('./config/custom.css').onModify(() => {
-	LoginServer.request('invalidatecss');
+	LoginServer.request('invalidatecss').catch(err => {
```

Comment:
catching and rethrowing does not useful other than trick the stupid linter. You want to `return` here and either `console.err` or ``// disable`` below

Reply:
To be clear, rethrowing does nothing. The whole point of the linter warning is to tell you that if `LoginServer.request('invalidatecss')` fails, nothing will noticed because the error is swallowed up. I